examples/json.pegjs

```diff
@@ -114,7 +114,7 @@ zero
 // ----- 7. Strings -----
 
 string "string"
-  = quotation_mark chars:char* quotation_mark { return chars.join(""); }
+  = quotation_mark @$char* quotation_mark
```

Comment:
That is change in the code semantics if we have an escape sequences in the input.

Reply:
fixed.